set.mm

```diff
@@ -677950,6 +677950,84 @@ collection and union ( ~ mnuop3d ), from which closure under pairing
+          -> E. u e. f ( i e. u /\ U. u C_ w ) ) ) ) ) $=
+  ( vg cv cpw cin wss cuni wa wrex wral wcel wi wal wex wel simpl reximi ralimi
+  wtru c0 0elpw a1i wceq biidd rspcdv mptru inss1 sstr2 mpi syl rexex ax5e 3syl
+  inex2 elpw mpbir unieq ineq2d ineq1 unieqd sseq12d anbi2d rexbidv rspcv ax-mp
+  vex alrimiv inss2 an12 elin bicomi anbi2i bitri exbii eluni 3bitr4i w3a simp1
+  df-rex biimpri 3adant1 sseldd sylib elinel1 elin2d elinel2 cvv elpwpw simprbi
+  anim2i eximi sylibr 3expia syl5bi ralrimiva anim12i sylg elequ2 rexeq imbi12d
+  jca ralbidv cbvalvw nfv nfa1 nfan elpwi sp ssin biimpi ex adantr simp3 simpl2
+  simprr simprl 3jca eximdv 3anass bitr4i mpbiran imim12d ralimdv imbi1i impexp
+  mpd albii dfss2 df-ral syl6ibr anim12d reximdv 3impia 3com23 syl3an2 ralrimia
+  3expa sylan2 impbid1 ralbiia ) AIZJZEBIZKZLZUUGFIZMZKZUULUUIJJZKZMZLZNZBEOZFE
+  JZPZUUHELZUUHUUILZGCUAZCIZUULQZNZCEOZGDUAZDIZMUUILZNZDUULOZRZGUUGPZNZBEOZFSZN
+  ZAEUUGEQZUVBUVTUVBUVTRUWAUVBUVCUVSUVBUUKBEOZFUVAPZUVCBEOZUVCUUTUWBFUVAUUSUUKB
+  EUUKUURUBUCUDUWCUWBUWDUWCUWBRUEUWBUWBFUFUVAUFUVAQUEEUGUHUEUULUFUINUWBUJUKULUU
+  KUVCBEUUKUUJELUVCEUUIUMUUHUUJEUNUOUCUPUWDUVCBTUVCUVCBEUQUVCBURUPUSUVBUVDUVEUV
+  FHIZQZNZCEOZUVMDUWEOZRZGUUGPZNZBEOZHSUVSUVBUUKUUGEUWEKZMZKZUWNUUOKZMZLZNZBEOZ
+  UWMHUVBUXAHUWNUVAQZUVBUXARUXBUWNELEUWEUMUWNEUWEEHVLUTVAVBUUTUXAFUWNUVAUULUWNU
+  IZUUSUWTBEUXCUURUWSUUKUXCUUNUWPUUQUWRUXCUUMUWOUUGUULUWNVCVDUXCUUPUWQUULUWNUUO
+  VEVFVGVHVIVJVKVMUWTUWLBEUUKUVDUWSUWKUUKUUJUUILUVDEUUIVNUUHUUJUUIUNUOUWSUWJGUU
+  GUWHGIZUWOQZUWSUXDUUGQZNUWIUVFEQZUWGNZCTUVEUVFUWNQZNZCTUWHUXEUXHUXJCUXHUVEUXG
+  UWFNZNUXJUXGUVEUWFVOUXKUXIUVEUXIUXKUVFEUWEVPVQVRVSVTUWGCEWECUXDUWNWAWBUWSUXFU
+  XEUWIUWSUXFUXEWCZUVJUVKUWQQZNZDTZUWIUXLUXDUWRQUXOUXLUWPUWRUXDUWSUXFUXEWDUXFUX
+  EUXDUWPQZUWSUXPUXFUXENUXDUUGUWOVPWFWGWHDUXDUWQWAWIUXOUVKUWEQZUVMNZDTUWIUXNUXR
+  DUXNUVJUXQUVLNZNUXRUXMUXSUVJUXMUXQUVLUXMEUWEUVKUVKUWNUUOWJWKUXMUVKUUOQZUVLUVK
+  UWNUUOWLUXTUVKWMQZUVLUVKUUIWNZWOUPXGWPUVJUXQUVLVOWIWQUVMDUWEWEWRUPWSWTXAXBUCX
+  CUVRUWMFHUULUWEUIZUVQUWLBEUYCUVPUWKUVDUYCUVOUWJGUUGUYCUVIUWHUVNUWIUYCUVHUWGCE
+  UYCUVGUWFUVEFHCXDVHVIUVMDUULUWEXEXFXHVHVIXIWRXGUHUVTUUTFUVAUVCUVSFUVCFXJUVRFX
+  KXLUULUVAQUVTUULELZUUTUULEXMUVCUVSUYDUUTUVSUVCUVRUYDUUTUVRFXNUVCUYDUVRUUTUVCU
+  YDUVRUUTUVCUYDNZUVQUUSBEUYEUVDUUKUVPUURUVCUVDUUKRUYDUVCUVDUUKUVCUVDNUUKUUHEUU
+  IXOXPXQXRUYEUVPUXDUUMQZUXDUUQQZRZGUUGPZUURUYEUVOUYHGUUGUYEUYFUVIUVNUYGUVCUYDU
+  YFUVIUVCUYDUYFWCZUXGUVEUVGWCZCTZUVIUYJUVHCTZUYLUYJUYFUYMUVCUYDUYFXSCUXDUULWAW
+  IUYJUVHUYKCUYJUVHUYKUYJUVHNZUXGUVEUVGUYNUULEUVFUVCUYDUYFUVHXTUYJUVEUVGYAZWHUY
+  JUVEUVGYBUYOYCXQYDYLUVIUXGUVHNZCTUYLUVHCEWEUYKUYPCUXGUVEUVGYEVTYFWRWSUVNUYGRU
+  YEUYGUVNUVJUVKUUPQZNZDTUVKUULQZUVMNZDTUYGUVNUYRUYTDUYRUVJUYSUVLNZNUYTUYQVUAUV
+  JUYQUYSUXTNVUAUVKUULUUOVPUXTUVLUYSUXTUYAUVLDVLUYBYGVRVSVRUYSUVJUVLVOYFVTDUXDU
+  UPWAUVMDUULWEWBWFUHYHYIUXDUUNQZUYGRZGSUXFUYHRZGSUURUYIVUCVUDGVUCUXFUYFNZUYGRV
+  UDVUBVUEUYGUXDUUGUUMVPYJUXFUYFUYGYKVSYMGUUNUUQYNUYHGUUGYOWBYPYQYRYSYTUUAUUCUU
+  DUUBUUEUUF $.
+  $}
```

Comment:
It seems that this proof is not properly formatted: it should be indented by two positions. I wonder why the checks did not detect that.

Reply:
The script in CONTRIBUTING.md under "Formatting recommendation prior to submitting a pull request" automatically indents the proof that way. Also, there are a few other proofs elsewhere in set.mm that have that kind of indentation.

Reply:
With the new commit, it's properly formatted. It seems that the formatting by metamath.exe was confused by the line break after `$p`, which I remember having seen already.